app/models/user.rb

```diff
@@ -1,5 +1,8 @@
 class User < ActiveRecord::Base
-  has_secure_password
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :trackable, :validatable
```

Comment:
Looks like devise gives us a lot o stuff.  Would love to hear more about it.  Also, I did a bit a research into Active Admin for the_pivot and remember devise was a big part of that gem.